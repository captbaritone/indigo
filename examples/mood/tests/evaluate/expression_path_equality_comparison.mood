enum Foo {
  A,
  B,
}

pub fn test(): bool {
  Foo::A == Foo::B
}