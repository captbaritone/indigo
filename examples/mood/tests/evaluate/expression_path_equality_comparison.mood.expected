-----------------
INPUT
----------------- 
enum Foo {
  A,
  B,
}

pub fn test(): bool {
  Foo::A == Foo::B
}
-----------------
OUTPUT
-----------------
TODO: Implement enum equality